### PR TITLE
fix: Log IMAP literals with append

### PR DESCRIPTION
**Pull request by LBeernaertProton**
We can now inspect the full message on append.

### CODE REVIEW

File: internal/session/command.go [hunk 1]
```diff
@@ -62,7 +65,11 @@ func (s *Session) readCommand(del string) (string, *proto.Command, error) {
 		return "", nil, err
 	}
 
-	s.logIncoming(string(line))
+	s.logIncoming(fmt.Sprintf("%v Literals:%v", string(line),
```
[the remark this answers is not in the document]
Comment (jameshoulahan) [reply internal/session/command.go:68]: or at least, don't print `Literals:%v` if literals is empty

[the remark this answers is not in the document]
Comment (LBeernaertProton) [reply internal/session/command.go:68]: it's faster to log than to perform replacements.

[the remark this answers is not in the document]
Comment (jameshoulahan) [reply internal/session/command.go:68]: I would then just not print the `Literals:%v` if there's nothing there (saves having an empty `Literals:` at the end of each log line)

and please add a space between `Literals:` and `%v`

[the remark this answers is not in the document]
Comment (LBeernaertProton) [reply internal/session/command.go]: Done

**Review by jameshoulahan — APPROVED** [review]

**Pull request merged**